Makefile

```diff
@@ -40,84 +40,57 @@ cri_version   =  release-1.26
 # Configuration Options
 export GIT_PAGER = cat
 
+#------------------------------------------------------------------------------#
+
+# Convenience commands
 
 .PHONY: all
 all: install ## Build and install (debug) 🎉
 
-.PHONY: install
-install: build ## Build and install (debug) 🎉
-
 .PHONY: build
-build: musl auraed auraescript ## Build and install (debug) (+musl) 🎉
+build: install ## Build and install (debug) (+musl) 🎉
 
-.PHONY: prcheck
-prcheck: build lint test-all ## Meant to mimic the GHA checks (includes ignored tests)
+#------------------------------------------------------------------------------#
 
-.PHONY: lint
-lint: ## Lint the code
-	@$(cargo) clippy -p auraed --target $(uname_m)-unknown-linux-musl --all-features -- -D clippy::all -D warnings
-	@$(cargo) clippy -p auraescript --all-features -- -D clippy::all -D warnings
-	@$(cargo) clippy -p aer --all-features -- -D clippy::all -D warnings
-	@$(cargo) clippy -p aurae-client --all-features -- -D clippy::all -D warnings
+# Super commands
 
-.PHONY: release
-release: proto## Build (static+musl) and install (release) 🎉
-	$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./auraed
-	$(cargo) install --path ./auraescript
+.PHONY: clean
+clean: clean-certs clean-gens clean-crates ## Clean the repo
 
-.PHONY: auraescript
-auraescript: proto ## Initialize and compile auraescript
-	@$(cargo) clippy -p auraescript
-	@$(cargo) install --path ./auraescript --debug --force
+.PHONY: lint
+lint: musl proto proto-lint libs-lint auraed-lint auraescript-lint aer-lint ## Run all lints
 
-.PHONY: aer
-aer: proto ## Initialize and compile aer
-	@$(cargo) clippy -p aer
-	@$(cargo) install --path ./aer --debug --force
+.PHONY: test
+test: lint libs-test auraed-test auraescript-test aer-test ## Run lints and tests (does not include ignored tests)
 
-musl: ## Add target for musl
-	rustup target add $(uname_m)-unknown-linux-musl
+.PHONY: test-all
+test-all: lint libs-test-all auraed-test-all auraescript-test-all aer-test-all ## Run lints and tests (includes ignored tests)
 
-.PHONY: auraed
-auraed: musl proto ## Initialize and static-compile auraed with musl
-	@$(cargo) clippy --target $(uname_m)-unknown-linux-musl -p auraed
-	@$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./auraed --debug --force
+.PHONY: install
```

Comment:
why is `install` debug but `release` release?  why does `install` not lint and test before building?

Reply:
The missing lint was an error in my refactoring. Should be fixed now.

Why it doesn't test boils down to not breaking peoples workflow. We used to have a `default` which was an alias for `all` which was an alias for `install`, so running `make` would lint and install auraed and auraescript (I have added aer now too), but not test.

We no longer have the default, so maybe my attempt at not effecting anyone's habits is moot?

Reply:
I updated `build` to not include tests (only lints) and `install` now has all three (lints, tests and install)